eng/common/scripts/Package-Properties.ps1

```diff
@@ -78,41 +78,35 @@ function Get-PkgProperties
     (
         [Parameter(Mandatory = $true)]
         [string]$PackageName,
-        [Parameter(Mandatory = $true)]
         [string]$ServiceDirectory
     )
 
-    $pkgDirectoryPath = $null
-    $serviceDirectoryPath = Join-Path $RepoRoot "sdk" $ServiceDirectory
-    if (!(Test-Path $serviceDirectoryPath))
-    {
-        LogError "Service Directory $ServiceDirectory does not exist"
-        return $null
-    }
-
-    $directoriesPresent = Get-ChildItem $serviceDirectoryPath -Directory
-
-    foreach ($directory in $directoriesPresent)
+    if ($ServiceDirectory)
     {
-        $pkgDirectoryPath = Join-Path $serviceDirectoryPath $directory.Name
-
-        if ($GetPackageInfoFromRepoFn -and (Test-Path "Function:$GetPackageInfoFromRepoFn"))
+        if (Test-Path $ServiceDirectory)
         {
-            $pkgProps = &$GetPackageInfoFromRepoFn -pkgPath $pkgDirectoryPath -serviceDirectory $ServiceDirectory -pkgName $PackageName
+            $ServiceDirectoryName = Split-Path $ServiceDirectory -Leaf
```

Comment:
Given that we use the name ServiceDirectory pretty much everywhere to mean just the name of the service directory I think we should just keep using the name ServiceDirectory instead of ServiceDirectoryName.